executor/explainfor_test.go

```diff
@@ -185,3 +185,21 @@ func (s *testPrepareSerialSuite) TestExplainForConnPlanCache(c *C) {
 		"  └─TableFullScan_5 10000.00 cop[tikv] table:t keep order:false, stats:pseudo",
 	))
 }
+
+func (s *testPrepareSerialSuite) TestExplainDotForExplainPlan(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	rows := tk.MustQuery("select connection_id()").Rows()
+	c.Assert(len(rows), Equals, 1)
+	connID := rows[0][0].(string)
+	tk.MustQuery("explain select 1").Check(testkit.Rows(
```

Comment:
```suggestion
	tk.MustQuery("explain format="dot" select 1").Check(testkit.Rows(
```

Reply:
great thanks for catch it 
😭

Reply:
it seems that the test is unchanged.

Reply:
> it seems that the test is unchanged.

Yes, actually, in my original vision, the first one does not need to be added(it is just an explain), and the second one needs to be added.

So I just fix the second explain.